src/leona/core.clj

```diff
@@ -186,17 +186,50 @@
 
 ;;;;;
 
+(defn transform-input-object-key [k]
+  (-> k
+      name
+      (str "_input")
+      keyword))
+
+(defn transform-input-object-keys [m]
+  (->> m
+       (map (fn [[k v]]
+              [(transform-input-object-key k)
+               v]))
+       (into {})))
+
+(defn replace-input-objects [m input-objects]
```

Comment:
This one's hardly legible. Could you add a docstring? And maybe give the inner fn a meaningful name, so we don't have these nested `postwalk`s in anonymous fns?